### DISCUSSION TITLE
Where can I find the ORIGINAL edge technology repo? And all the voices available?

### DISCUSSION THREAD

**Discussion by APISeeker** [rany2/edge-tts#193] [Q&A] [answered]
Hello
I am using a tech related to edge, and it has a certain number of voices, but I am not sure I have the whole list, where can i find all the info related to the original project? And find ALL the actual voices actually available and offered etc?

Thanks

**Comment by rany2** [comment, marked as answer]
https://github.com/rany2/edge-tts/issues/192
